azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/b2c/AADB2CEntryPoint.java

```diff
@@ -28,9 +29,17 @@ public AADB2CEntryPoint(@NonNull AADB2CProperties properties) {
     }
 
     @Override
-    public void commence(HttpServletRequest request, HttpServletResponse response, AuthenticationException e) throws
-            IOException {
-        final String requestURL = request.getRequestURL().toString();
+    public void commence(HttpServletRequest request, HttpServletResponse response, AuthenticationException e)
+            throws IOException {
+        String requestURL = request.getRequestURL().toString();
+        final URL url = new URL(requestURL);
+
+        // By default, request URL will be '/error' when exception handling, here handle this endpoint to
+        // policy redirect URL instead of redirect to '/error' directly.
+        if (url.getPath().equalsIgnoreCase("/error")) {
```

Comment:
I am okay with POC here. But for the final solution, I think we need more discussion.

Reply:
Yes, absolutely.